packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -225,6 +225,9 @@ Resources:
               - Ref: AWS::NoValue
       RoleName:
         Fn::Sub: cdk-${Qualifier}-file-publishing-role-${AWS::AccountId}-${AWS::Region}
+      Tags:
+        - Key: aws-cdk:bootstrap-role
+          Value: file-publishing
```

Comment:
Open to advice on where we'd document this convention such that folks can now override these role names as long as they tag consistently.

Reply:
Needs to go here I think

https://docs.aws.amazon.com/cdk/latest/guide/bootstrapping.html